action.yml

```
@@ -226,12 +233,22 @@ runs:
         inputs.pull-request-change-detection
       }}')
       pull_request_branch = '${{ github.event.pull_request.base.ref || '' }}'
+      coverage = '${{ inputs.coverage }}'
+
+      # Validate GHA inputs
+      if coverage not in ('always', 'never', 'auto'):
```

It is semantically better to use sets for such checks
```suggestion
      if coverage not in {'always', 'never', 'auto'}:
```